daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -934,18 +934,12 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite) {
         _.nBits
       }
 
-      val useSerializedProcessor =
-        if (validation == "on") false
-        else if (defaultValidation == "on") false
-        else if (optExpectedWarnings.isDefined) false
-        else true
-
       val rootNamespaceString = getRootNamespaceString()
 
       val compileResult: TDML.CompileResult = parent.getCompileResult(
         impl,
         suppliedSchema,
-        useSerializedProcessor,
+        useSerializedProcessor = true,
```

Comment:
Can we completely remove all occurrences of useSerailizedProcessor? This is now always true and we always use a serialized processor, so I don't think it should be needed anymore.